### PR TITLE
Ldca 413

### PR DESCRIPTION
**Pull request by najhiullah**
Move Senion Android from SWP to eeGeo Example App

### CODE REVIEW

File: android/AndroidManifest.xml [hunk 1]
```diff
@@ -13,11 +13,23 @@
...
     <uses-permission android:name="android.permission.CAMERA"/>
-    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
-    <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
-    <uses-feature android:name="android.hardware.camera"
-        android:required="true"/>
-
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
+    <uses-permission android:name="android.permission.ACCESS_WIFI_STATE"/>
+    <uses-permission android:name="android.permission.CHANGE_WIFI_STATE" />
+	<uses-permission android:name="android.permission.READ_PHONE_STATE" />
```
Comment (tim-jenks) [inline android/AndroidManifest.xml:19]: Required?

File: android/AndroidManifest.xml [hunk 2]
```diff
@@ -13,11 +13,23 @@
...
+    <uses-permission android:name="android.permission.ACCESS_COARSE_UPDATE" />
+    <uses-permission android:name="android.permission.BLUETOOTH" />
+    <uses-permission android:name="android.permission.BLUETOOTH_ADMIN" />
+    <uses-permission android:name="android.permission.VIBRATE" />
```
Comment (tim-jenks) [inline android/AndroidManifest.xml]: Required?

File: android/AndroidManifest.xml [hunk 3]
```diff
@@ -13,11 +13,23 @@
...
+   	<uses-permission android:name="android.permission.WAKE_LOCK" />
```
Comment (tim-jenks) [inline android/AndroidManifest.xml]: Required?

File: android/jni/AppHost.h [hunk 4]
```diff
@@ -117,6 +121,10 @@ class AppHost : public Eegeo::IEegeoErrorHandler, protected Eegeo::NonCopyable
 
     ExampleApp::ModalBackground::SdkModel::IModalBackgroundNativeViewModule* m_pModalBackgroundNativeViewModule;
 
+    ExampleApp::IndoorAtlas::IndoorAtlasLocationModule* m_pIndoorAtlasLocationModule;
```
Comment (tim-jenks) [inline android/jni/AppHost.h:125]: Can we move IA and Senion to ExampleApp::IPS:: to prevent namespace pollution

**Review by tim-jenks — CHANGES_REQUESTED** [review]
Only a partial review.

1. Please remove redundant 'avatar' assets on Android, may also exist on IOS

2. Please move Indoor Positioning Types to ExampleApp::IPS:: namespace

Will review again more thoroughly when this change is an isolated Senion change, not interleaved with the IndoorAtlas change.

This change seems to contain both significant additions to IndoorAtlas and Senion. Suggest we submit a single Senion PR when IndoorAtlas has been accepted.

**Comment by SirPryderi** [comment]
Outdated -- closing.

**Pull request closed without merging**